css/index.css

```diff
@@ -717,14 +763,165 @@ input:-webkit-autofill:active  {
     background-color: whitesmoke;
 }
 
+.modal .form{
+    /*Sizing*/
+    width: 100%;
+}
+
 
 .modal h1{
     /* Margins */
-    margin: 0;
+    margin-top: 2px;
+    margin-bottom: 0px
 }
 
 .modal-container.show {
     /* Display */
     visibility: visible;
     opacity: 1;
 }
+
+/* 
+==================================
+=          MEDIA QUERIES         =
+==================================
+*/
+
+/* TABLETS */
+/* MAXIMUM TABLET WIDTH OF 1024PX*/
+@media (max-width: 1024px){
```

Comment:
Great use of media queries, especially in the nav bar.

Reply:
Thank you!